.gitignore

```diff
@@ -7,6 +7,7 @@ node_modules
 .nyc_output
 .rpt2_cache
 .vscode
+.idea
```

Comment:
I omitted this line, but FWIW you can put this into a `.gitignore` file in your user directory (e.g. `~/.gitignore`) and it should apply to all projects you work on. This makes it easier to contribute to multiple projects without having to customize `.gitignore` for each.

Reply:
Hey thanks for the tip!! I should have simply omitted this change, but I will keep that in mind for the future. Cheers!